src/App.js

```diff
@@ -1,10 +1,12 @@
 import "antd/dist/antd.css";
```

Comment:
first all external imports then 1 space, then internal functions and component import, then 1 space and finally css imports should occur in the end

Reply:
sir isn't this an external import?

Reply:
anything that is imported from installed libraries and not created by you

Reply:
but it's a CSS file so it'll come in the end along with normal internal css